### ISSUE TITLE
MIS 4 Notation - primitive data types

### ISSUE THREAD

**Issue by peter-michalski** [sorainsm/library-of-lighting-models#48]
I am unsure if some of the data types listed in the Section 4 table should be there. The section describes the table as a table for primitive data types, but primitive data types are those basic data types provided by the programming language or those built-in by the language for which the language has support. 

Some of the listed data types in your table are simply created in your program (such as Mesh) which are abstract data types, not primitive data types. 

You could remove them, or change the description of the table.

I would like @smiths to confirm.

**Comment by smiths** [comment]
Yes, I agree with @peter-michalski.  Point3D, Colour, Shape etc are not primitive data types.  These types should be defined in the MIS.  If they do not have behaviour, they can be implemented as record types.  If they have behaviour, then they would be shown via an ADT.
